From 59031299cbfba766f9197c46f5271b4cb4afb89d Mon Sep 17 00:00:00 2001
From: Andrea Adami <andrea.adami@gmail.com>
Date: Wed, 17 Jul 2013 00:27:23 +0200
Subject: [PATCH] tosa_battery.c: get rid of irq_to_gpio usage

Fix
linux/drivers/power/tosa_battery.c:153:2: error: implicit declaration of
function 'irq_to_gpio' [-Werror=implicit-function-declaration]
|   pr_info("tosa_bat_gpio irq: %d\n",
gpio_get_value(irq_to_gpio(irq)));

as done for collie_battery.c with
commit 629bcb4b72d49b3631ae3dd0fe1d345820fadfcc

Since 9d08d5d77a355510c2f5657c86b0a4b25acfe72c, irq_to_gpio() is no
longer available but still in use by collie_battery.c. As it's just
for a debug message, just get rid of this call.

Signed-off-by: Andrea Adami <andrea.adami@gmail.com>
---
 drivers/power/tosa_battery.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/tosa_battery.c b/drivers/power/tosa_battery.c
index 0224de5..f4d80df 100644
--- a/drivers/power/tosa_battery.c
+++ b/drivers/power/tosa_battery.c
@@ -150,7 +150,7 @@ static void tosa_bat_external_power_changed(struct power_supply *psy)
 
 static irqreturn_t tosa_bat_gpio_isr(int irq, void *data)
 {
-	pr_info("tosa_bat_gpio irq: %d\n", gpio_get_value(irq_to_gpio(irq)));
+	pr_info("tosa_bat_gpio irq\n");
 	schedule_work(&bat_work);
 	return IRQ_HANDLED;
 }
-- 
1.8.1.5

